README.md

```diff
@@ -1079,7 +1079,7 @@ We will be writing our own bash script that will involve the use of some of the
 
   [Gulp](https://github.com/gulpjs/gulp/blob/master/docs/API.md) can be used to automate the zipping, deployment and testing of Lambda functions on AWS. The Codeship deployment script can then be reduced to a single command `gulp deploy`!
 
-  The syntax to create a new Gulp task is"
+  The syntax to create a new Gulp task is:
```

Comment:
This is a straightforward typo fix. After this, the changes are all for MarkDown formatting, and look boring/ugly in plaintext source mode. But they do improve the readability.